dts/common/nordic/nrf54h20.dtsi

```diff
@@ -846,13 +850,16 @@
 				#size-cells = <0>;
 				rx-delay-supported;
 				rx-delay = <1>;
+				nordic,clock-enable = <NRF_FUN_SPIM_MOSI>,
+						      <NRF_FUN_SPIM_SCK>;
```

Comment:
+ NRF_FUN_SPIS_MISO and NRF_FUN_SPIS_SCK

Reply:
spi131 is missing the entry.

Reply:
done